TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/fragment/PostFragment.java

```diff
@@ -219,14 +234,42 @@ private void setPostButton(View view) {
         });
     }
 
+    private void setUserButton(View view) {
+        // Get user
+        TwitterUser user = TweetMateApp.getMyUser();
+        ImageView userButton = view.findViewById(R.id.button_user);
+
+        // Set user image
+        ImageOption option = ImageOption.toEnum(PrefAppearance.getUserIconStyle());
+        GlideUtils.load(PrefSystem.getProfileThumbByQuality(user), userButton, option);
+
+        // User button
+        userButton.findViewById(R.id.button_user).setOnClickListener(v -> {
+            // Coming soon...
+        });
+    }
+
+    private void setGalleryButton(View view) {
+        // Gallery button
+        view.findViewById(R.id.button_gallery).setOnClickListener(v -> {
+            if (!isPicAvailable()) {
+                return;
+            }
+            // Open gallery
+            Intent intent = new Intent(Intent.ACTION_PICK);
+            intent.setType("image/*");
+            getActivity().startActivityForResult(intent, REQUEST_GALLERY);
+        });
+    }
+
     private void setDraftButton(View view) {
         // Draft button
         view.findViewById(R.id.button_draft).setOnClickListener(v -> {
 
             // Check drafts
             EntityArray<String> drafts = TweetMateApp.getMyAccount().getDrafts();
             if (drafts.isEmpty()) {
-                NoticeDialog dialog = new NoticeDialog().newInstance("下書きはありません。");
+                NoticeDialog dialog = new NoticeDialog().newInstance(STR_FAIL_NO_DRAFT);
                 FragmentManager fm = getActivity().getSupportFragmentManager();
                 if (fm.findFragmentByTag("NoticeDialog") == null) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "NoticeDialog" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1192)

Reply:
Pending